app/rest.py

```diff
@@ -2,3 +2,5 @@
 
 if __name__ == "__main__":
     app.run(debug=True, host="0.0.0.0")
+
+import journals
```

Comment:
what is journals, should this not be articles?

Reply:
Makes sense Dom, will make the change

Reply:
These changes were also made to the database schema